webapp/src/components/backstage/playbook_edit/automation/channel_access.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+import {useIntl} from 'react-intl';
+
+import {DraftPlaybookWithChecklist, PlaybookWithChecklist} from 'src/types/playbook';
+import {PatternedInput} from 'src/components/backstage/playbook_edit/automation/patterned_input';
+import {AutomationHeader, AutomationTitle} from 'src/components/backstage/playbook_edit/automation/styles';
+import {Toggle} from 'src/components/backstage/playbook_edit/automation/toggle';
+import {RadioInput} from 'src/components/backstage/styles';
+import {HorizontalSpacer} from 'src/components/backstage/playbook_runs/shared';
+
+interface Props {
+    playbook: DraftPlaybookWithChecklist | PlaybookWithChecklist;
+    setPlaybook: (playbook: DraftPlaybookWithChecklist | PlaybookWithChecklist) => void;
+    setChangesMade: (b: boolean) => void;
+}
+
+export const CreateAChannel = ({playbook, setPlaybook, setChangesMade}: Props) => {
+    const {formatMessage} = useIntl();
+
+    const handlePublicChange = (isPublic: boolean) => {
+        setPlaybook({
+            ...playbook,
+            create_public_playbook_run: isPublic,
+        });
+        setChangesMade(true);
+    };
+
+    const handleChannelNameTemplateChange = (channelNameTemplate: string) => {
+        setPlaybook({
+            ...playbook,
+            channel_name_template: channelNameTemplate,
+        });
+        setChangesMade(true);
+    };
+
+    return (
+        <AutomationHeader>
+            <AutomationTitle>
+                <Toggle
+                    isChecked={true}
+                    disabled={true}
+                    onChange={() => null}
+                />
+                <div>{formatMessage({defaultMessage: 'Create a channel'})}</div>
+            </AutomationTitle>
+            <HorizontalSplit>
+                <VerticalSplit>
+                    <Button onClick={() => handlePublicChange(true)}>
+                        <RadioInput
+                            type='radio'
+                            checked={playbook.create_public_playbook_run}
+                        />
+                        <Icon
+                            active={playbook.create_public_playbook_run}
+                            className={'icon-globe'}
+                        />
+                        <BigText>{'Public'}</BigText>
+                    </Button>
```

Comment:
These two should probably be wrapped in labels instead of buttons, they're naturally clickable and don't interfere with tab focus indexing.

I18n, also

```suggestion
                    <ButtonLabel>
                        <RadioInput
                            type='radio'
                            checked={playbook.create_public_playbook_run}
                            onChange={() => handlePublicChange(true)}
                        />
                        <Icon
                            active={playbook.create_public_playbook_run}
                            className={'icon-globe'}
                        />
                        <BigText>{formatMessage({defaultMessage: 'Public'})}</BigText>
                    </ButtonLabel>
```